jumpscale/data/serializers/toml.py

```diff
@@ -0,0 +1,111 @@
+import pytoml
+def fancydumps(self, obj, secure=False):
```

Comment:
Please remove this fancy code that won't work.

Reply:
solved in the last commit